app/migrations/Version20151221151112.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Sylius\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20151221151112 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('DROP INDEX fulltext_search_idx ON sylius_search_index');
+        $this->addSql('CREATE INDEX fulltext_search_idx ON sylius_search_index (item_id)');
+        $this->addSql('ALTER TABLE sylius_product_variant CHANGE available_to available_until DATETIME DEFAULT NULL');
```

Comment:
I think you don't need this migration when you have the other one. Just change `available_to` to `available_util` in previous migration.